cmd/esc/cli/login.go

```diff
@@ -172,14 +172,8 @@ func (esc *escCommand) getCachedClient(ctx context.Context) error {
 			return fmt.Errorf("could not determine current cloud: %w", err)
 		}
 
-		defaultOrg, err := esc.workspace.GetBackendConfigDefaultOrg(backendURL, nAccount.Username)
-		if err != nil {
-			return fmt.Errorf("could not determine default org: %w", err)
-		}
-
 		account = &workspace.Account{
-			Account:    *nAccount,
-			DefaultOrg: defaultOrg,
```

Comment:
From reading the code, it seems like this (and L212-216) are both safe to remove.

`defaultOrg` configuration appears to get overridden later by `getCachedCredentials`, anyway. I've moved the default org setting to be after the client is initialized.